### ISSUE TITLE
Some default shortcuts are not compatible with mac laptops 

### ISSUE THREAD

**Issue by artisteacher** [opentoonz/opentoonz#1327]
My macbook's keyboard does not have a numberpad and my only delete key is the same as the backspace key on other keyboards. This means that with the default shortcuts, I can't use delete. Any shortcuts with a + (such as zoom in or fullscreen) also aren't possible (my + key is part of the = key).

 While I've saved my own shortcuts, I think that the default shortcuts should work for mac laptops. In the same way that the command and control keys are treated as the same key, backspace and delete could be treated as the same key. The same for the numberpad's + key and the = key.

**Comment by artisteacher** [comment]
For anyone else on a Mac laptop, using fn-delete works like delete should without changing the shortcut. 

I could create a custom set of Mac laptop keyboard shortcuts to share as a workaround, but it would still be lovely to get backspace/delete treated as the same key and for +/= to be treated as the same key.

**Comment by MrBlockCat** [comment]
I agree something should be done about that, having a MacBook with a compact keyboard myself, though of course it'd be nice to figure out where and how Control is converted to Command and Alt to Option on macOS so that that same code could be used for Insert to (?), Backspace to Delete, and '+' to Shift + '='!

**Comment by MrBlockCat** [comment]
That said, what should the Ins key be replaced with? We’d need to figure that out both for a long term (macOS detection and conversion of keys) and short term (macOS preset) fix of this issue
